lib/iris/cube.py

```diff
@@ -697,6 +695,14 @@ def __init__(self, data, standard_name=None, long_name=None,
             :mod:`iris.aux_factory`.
         * cell_measures_and_dims
             A list of CellMeasures with dimension mappings.
+        * fill_value
+            The intended fill-value of :class:`~iris.cube.Cube` masked data.
+            Note that, the fill-value is cast relative to the dtype of the
+            :class:`~iris.cube.Cube`.
+        * dtype
+            The intended dtype of the specified lazy data, which must be
+            either integer or boolean. This is to handle the case of lazy
+            integer or boolean masked data.
```

Comment:
I wonder if this description could be clearer. I have no wish to hold this PR up any further though, so I'll think about it in slow time and perhaps push up something new that helps this description out!